### PR TITLE
Font/SVG Documentation + Global Preprocessor Imports

### PR DESCRIPTION
**Pull request by K3TH3R**
This PR has some documentation about using Font/SVGs with Gridsome. I also added a block to the CSS docs on how to use Global Preprocessor Imports so that you don't have to always manually include the same files for stuff like variables, mixins, etc.

### CODE REVIEW

**Comment by K3TH3R** [comment]
@tomtev that sounds like a good idea, I'll whip that up tomorrow

**Comment by K3TH3R** [comment]
@tomtev I think we're all good here now

File: docs/assets-svg.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+  }
+}
+```
+
+Then you can import your SVGs from within your Vue templates like any normal Vue component:
+```html
+<script>
+  import Brand from '~/assets/svgs/Brand.svg'
+  export default {
+    compoents: Brand
```
Comment (physcocode) [inline docs/assets-svg.md]: *Component

File: docs/assets-svg.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+  }
+</script>
+```
+
+# FontAwesome SVG Icons
+
+FontAwesome is one of the most popular icon libraries out there, and with version 5, they've made it incredibly easy to use with Vue without bloating your codebase with a lot of icons/styles you're not using. We can start implementing it in our Gridsome site by installing a few needed items:
+
+```sh
+npm i -d @fortawesome/{vue-fontawesome,fontawesome-svg-core,free-brands-svg-icons}
```
Comment (physcocode) [inline docs/assets-svg.md]: *fontawesome

**Comment by tomtev** [comment]
That's perfect :) Could you just fix that little typo and I will merge it.

**Comment by K3TH3R** [comment]
@tomtev I added the documentation for my [Overriding Index.html PR](https://github.com/gridsome/gridsome/pull/162) here since this hasn't been merged in yet.

**Comment by tomtev** [comment]
@K3TH3R Thank you :) We'll merge this when 0.5 is ready

**Comment by tomtev** [comment]
Thank you so much for this :) We'll merge this now. 0.5 will be published in a couple of hours.

**Pull request merged**